book/src/builders.md

```diff
@@ -0,0 +1,45 @@
+# MEV and Lighthouse
+
+A primer on MEV can be found [here]([MEV](https://ethereum.org/en/developers/docs/mev/)).
+
+Lighthouse is able to interact with servers that implement the [builder API](https://github.com/ethereum/builder-specs). 
+If the builder you are connected to 
```

Comment:
These docs are a bit incomplete. If we tidy them up to be minimally correct (e.g. no incomplete sentences or wrong info) I think we can merge now and come back for more comprehensive docs later